webpack.config.js

```diff
@@ -26,6 +26,9 @@ const OUT_PATH = path.resolve('./build');
 const PUBLIC_PATH = '/assets/';
 const IS_DEV = process.env.MDC_ENV === 'development';
 const IS_PROD = process.env.MDC_ENV === 'production';
+const IS_FAST_BUILD = process.env.MDC_BUILD === 'fast';
+const GENERATE_SOURCE_MAPS = IS_DEV && !IS_FAST_BUILD && process.env.MDC_GENERATE_SOURCE_MAPS !== 'false';
```

Comment:
Is there a reason we're introducing multiple env vars here (`MDC_BUILD` and `MDC_GENERATE_SOURCE_MAPS` here, and `MDC_DEVTOOL` below)? Do you have more plans for `MDC_BUILD=fast`?

Do we document the env vars we look at here, anywhere? Should we (even if it's just in this file above the constants we define)?